### PR TITLE
davmail: 4.7.2 -> 4.8.0

### PR DESCRIPTION
**Pull request by tosmi**
###### Motivation for this change
Update davmail to version 4.8.0, running 4.8.0 locally on nixos 17.03.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X ] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by rasendubi — COMMENTED** [review]
Looks good.

The only nitpick is [nixpkgs guidelines](http://nixos.org/nixpkgs/manual/#idm140737318408704) require commit name to be `davmail: 4.7.2 -> 4.8.0`

**Comment by tosmi** [comment]
sorry for that. saw the guidelines to late. should i amend the commit message?

**Comment by tosmi** [comment]
i've already changed the commit message

File: pkgs/applications/networking/davmail/default.nix [hunk 1]
```diff
@@ -1,10 +1,10 @@
 { fetchurl, stdenv, jre, glib, libXtst, gtk2, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "davmail-4.7.2";
+  name = "davmail-4.8.0";
   src = fetchurl {
-    url = "mirror://sourceforge/davmail/4.7.1/davmail-linux-x86_64-4.7.1-2416.tgz";
-    sha256 = "196jr44kksb197biz984z664llf9z3d8rlnjm2iqcmgkjhx1mgy3";
+    url = "mirror://sourceforge/davmail/4.7.1/davmail-linux-x86_64-4.8.0-2479.tgz";
```
Comment (tosmi) [inline pkgs/applications/networking/davmail/default.nix]: oops, the mirror url seems to wrong, still points to 4.7.1. building works never the less as i'm running 4.8.0. i will change the url, re-test and rebase my commits.

**Comment by tosmi** [comment]
sorry for the late response, i've fixed the path to the download url now and clean up my commits.

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**